packages/harpparts/src/apparatus/utils/is-hole-valid/is-hole-valid.ts

```diff
@@ -0,0 +1,88 @@
+import type { Hole } from '../../types'
+import {
+  isConsecutiveWithPrevious,
+  Direction,
+} from '../../../packages/is-consecutive-with-previous'
+
+export enum HoleErrors {
+  ConflictingDrawBends = 'CONFLICTING_DRAW_BENDS',
```

Comment:
Should be `ConflictingDrawbends` for consistency.

Also the next one should be `ConflictingBlowbends`.

Reply:
Done